clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBServerConfig.java

```diff
@@ -338,9 +338,9 @@ public RocksDBServerConfig(VeniceProperties props) {
     }
 
     this.rocksDBBlockCacheSizeInBytes =
-        props.getSizeInBytes(ROCKSDB_BLOCK_CACHE_SIZE_IN_BYTES, 16 * 1024 * 1024 * 1024L); // 16GB
+        props.getSizeInBytes(ROCKSDB_BLOCK_CACHE_SIZE_IN_BYTES, 2 * 1024 * 1024 * 1024L); // 2GB
```

Comment:
why the jump from 16 -> 2?

Reply:
Glancing at our config repo, we're already explicitly setting these in all colos, so the only users who this 'might' impact at davinci users?  We should check that.  I don't think shrinking it is going to be a problem for server deployments.  Though I too am curious to the logic for why we'd like to shrink these defaults in this PR

Reply:
To make all the tests pass as mac jvm allocates 4G of memory

Reply:
DVC users pick up from global-config or can override it, which is also set to 16G.

Reply:
Can we please verify how this config is getting defaulted for Davinci users?  If we're covered there I think this is fine.

Reply:
Emm.. Why not just override it in `VeniceServerWrapper` instead of here to let test pass in Mac?